sdk/samples/iot/README.md

```diff
@@ -92,7 +92,7 @@ To run the samples, ensure you have the following programs and tools installed o
 
 - Have the following build environment setup:
 
-    <details><summary><i>Instructions:</i></summary>
+    <summary><i>Instructions:</i></summary>
```

Comment:
please remove this line. You can also remove the `<p>` tags surrounding the text. 

[I have an example here for how to indent the text to look like the following:](https://github.com/danewalton/azure-sdk-for-c/blob/57efa7d640b6ead4d9200dbe1e374fee6f86952e/sdk/samples/iot/README.md?plain=1#L93-L106)

![image](https://user-images.githubusercontent.com/36710865/207934210-1f3e77db-78ae-42d9-ae45-5e31cd3ec6c9.png)

Reply:
So it looks like we have to remove some instruction lines.

Reply:
Modified